includes/Modules/AdSense.php

```diff
@@ -202,36 +202,80 @@ public function on_deactivation() {
 	public function get_debug_fields() {
 		$settings = $this->get_settings()->get();
 
-		return array(
-			'adsense_account_id'            => array(
+		$fields = array(
+			'adsense_account_id'                => array(
 				'label' => __( 'AdSense account ID', 'google-site-kit' ),
 				'value' => $settings['accountID'],
 				'debug' => Debug_Data::redact_debug_value( $settings['accountID'], 7 ),
 			),
-			'adsense_client_id'             => array(
+			'adsense_client_id'                 => array(
 				'label' => __( 'AdSense client ID', 'google-site-kit' ),
 				'value' => $settings['clientID'],
 				'debug' => Debug_Data::redact_debug_value( $settings['clientID'], 10 ),
 			),
-			'adsense_account_status'        => array(
+			'adsense_account_status'            => array(
 				'label' => __( 'AdSense account status', 'google-site-kit' ),
 				'value' => $settings['accountStatus'],
 			),
-			'adsense_site_status'           => array(
+			'adsense_site_status'               => array(
 				'label' => __( 'AdSense site status', 'google-site-kit' ),
 				'value' => $settings['siteStatus'],
 			),
-			'adsense_use_snippet'           => array(
+			'adsense_use_snippet'               => array(
 				'label' => __( 'AdSense snippet placed', 'google-site-kit' ),
 				'value' => $settings['useSnippet'] ? __( 'Yes', 'google-site-kit' ) : __( 'No', 'google-site-kit' ),
 				'debug' => $settings['useSnippet'] ? 'yes' : 'no',
 			),
-			'adsense_web_stories_adunit_id' => array(
+			'adsense_web_stories_adunit_id'     => array(
 				'label' => __( 'Web Stories Ad Unit ID', 'google-site-kit' ),
 				'value' => $settings['webStoriesAdUnit'],
 				'debug' => $settings['webStoriesAdUnit'],
 			),
+			'adsense_setup_completed_timestamp' => array(
+				'label' => __( 'AdSense setup completed at', 'google-site-kit' ),
+				'value' => date_i18n(
+					get_option( 'date_format' ),
+					$settings['setupCompletedTimestamp']
+				),
```

Comment:
According to the AC, if the setting is empty, the value should be `Not available`.

>* Value: date formatted according to the site's locale. If the setting is empty, it should be `Not available`

Reply:
Addressed, thank you!